code/game/antagonist/station/revolutionary.dm

```diff
@@ -71,7 +71,7 @@ var/datum/antagonist/revolutionary/revs
 	return TRUE
 
 /datum/antagonist/revolutionary/proc/alert_rev_status(var/mob/living/carbon/human/player) //This is so dumb.
-	alert(player, "As a Head Revolutionary, you are given an uplink with a lot of telecrystals. \
+	alert(player, "As a Head Fellow, you are given an uplink with a lot of telecrystals. \
```

Comment:
```suggestion
	alert(player, "As a Head Contendor you are given an uplink with a lot of telecrystals. \
```
You are at the Contendors here.
You should add the same notice to the Fellowship Leader.

Reply:
Ok, both of these done and I have that alert for the Contendor too (I think)